dist/formats/finder/frontend/schema.json

```diff
@@ -428,10 +439,13 @@
             "description": "Defines the UI component and how the facet is queried from the search API",
             "type": "string",
             "enum": [
-              "text",
+              "autocomplete",
+              "checkbox",
               "date",
-              "topical",
-              "hidden"
+              "hidden",
+              "taxon",
+              "text",
+              "topical"
```

Comment:
Is 'topical' something we added?

Reply:
No, but it might be useful looking at the `TopicalFacet` in finder-frontend: https://github.com/alphagov/finder-frontend/commit/01fb3e7e63e7dc09024acd94c9c6a8c1f5f66c57#diff-45d0a10e840ba0a077119e9bc4bf54db